docs/toc.yml

```diff
@@ -269,16 +269,16 @@
         - name: Tutorial
           href: tutorials/excel-tutorial-create-custom-functions.md
           displayName: Excel
-        - name: Web requests
+        - name: Making web requests
           href: excel/custom-functions-web-reqs.md
           displayName: Excel
         - name: Runtime
           href: excel/custom-functions-runtime.md
           displayName: Excel
-        - name: Metadata
+        - name: JSON Metadata
           href: excel/custom-functions-json.md
           displayName: Excel
-        - name: Custom function metadata
+        - name: Autogenerate metadata
```

Comment:
Considering this article discusses manual steps the customer needs to do, maybe this title should be "Metadata generation"?